.github/workflows/build-os-image.yml

```diff
@@ -273,6 +274,7 @@ jobs:
           BOOTSTRAPPER_BINARY: ${{ github.workspace }}/build/bootstrapper
           DISK_MAPPER_BINARY: ${{ github.workspace }}/build/disk-mapper
           UPGRADE_AGENT_BINARY: ${{ github.workspace }}/build/upgrade-agent
+          AUTOLOGIN: ${{ inputs.stream == 'console' && 'true' || 'false' }}
```

Comment:
Can we also do this for 'debug' and then remove the debugd code that does this otherwise?

Reply:
Good idea! :+1: